features/dual-stack-support.md

```diff
@@ -0,0 +1,238 @@
+[//]: # (The syntax preceeding this line is a comment marker used to help guide the author in populating this document)
+[//]: # (to github. Unlike HTML comments commonly used throughout istio.io documentation, this comment will not be rendered)
+[//]: # (by github. Comments must be separated by carriage return preceding and concluding the text and be a single line.)
+
+[//]: # (This is a living document representing the maturity of a feature. Completion of this template enables Istio work groups)
+[//]: # (to collect information on potential new functionality. This template should be completed before users are exposed to)
+[//]: # (any new experimental feature. Please complete this template during development.)
+
+[//]: # (The feature implementation section must be completed before submission of the document.)
+
+# Feature:
+
+[//]: # (All information in this section is mandatory.)
+
+**Feature name:**
+
+Dual stack support in Istio
+
+[//]: # (The name of the feature, e.g. Multiple control planes)
+
+**Primary lead(s):**
+
+Steve Zhang (huailong.zhang@intel.com), Jacob Delgado (j.delgado@f5.com)
+
+[//]: # (The primary lead or leads responsible for the feature. These individuals serve as a point of contact for the feature.)
+
+
+**Short description:**
+
+Add support for Dual Stack network model in Istio. With this, besides both IPv6 and IPv4 network model, Istio now supports dual-stack at the moment.
```

Comment:
doesn't dual stack include both ipv4 and ipv6?

Reply:
Hi @hzxuzhonghu, I think there should be 3 models: IPv4 only, IPv6 only and Dual Stack. So I will change the statement as this.